spec/values/calculation/atan2.hrx

```diff
@@ -0,0 +1,204 @@
+a {b: atan2(1, -10)}
+
+<===> unitless/output.css
+a {
+  b: 174.2894068625deg;
+}
+
+<===>
+================================================================================
+<===> compatible_units/input.scss
+a {b: atan2(1cm, -10mm)}
+
+<===> compatible_units/output.css
+a {
+  b: 135deg;
+}
+
+<===>
+================================================================================
+<===> operation/math/input.scss
+a {b: atan2(10 + 6, 8 - 1)}
+
+<===> operation/math/output.css
+a {
+  b: 66.3706222693deg;
+}
+
+<===>
+================================================================================
+<===> operation/nested_function/input.scss
+a {b: atan2(max(2mm, 9mm), min(1cm, 3cm))}
+
+<===> operation/nested_function/output.css
+a {
+  b: 41.9872124958deg;
+}
+
+<===>
+================================================================================
+<===> operation/simplification/input.scss
+a {
+  b: atan2(3px - 1px + var(--c), 4px + 10px);
+}
+
+<===> operation/simplification/output.css
+a {
+  b: atan2(2px + var(--c), 14px);
+}
+
+<===>
+================================================================================
+<===> preserved/y_arg/percentage/input.scss
+a {b: atan2(1.5px + 5%, 4px)}
+
+<===> preserved/y_arg/percentage/output.css
+a {
+  b: atan2(1.5px + 5%, 4px);
+}
+
+<===>
+================================================================================
+<===> preserved/x_arg/percentage/input.scss
+a {b: atan2(4px, 1.5px + 5%)}
+
+<===> preserved/x_arg/percentage/output.css
+a {
+  b: atan2(4px, 1.5px + 5%);
+}
+
+<===>
+================================================================================
+<===> preserved/variable/input.scss
+a {
+  b: atan2(var(--c));
+}
+
+<===> preserved/variable/output.css
+a {
+  b: atan2(var(--c));
+}
+
+<===>
+================================================================================
+<===> preserved/unknown_units/input.scss
+a {
+  b: atan2(1foo, 2bar);
+}
+
+<===> preserved/unknown_units/output.css
+a {
+  b: atan2(1foo, 2bar);
+}
+
+<===>
+================================================================================
+<===> error/incompatible_units/input.scss
+a {b: atan2(1, 1px)}
+
+<===> error/incompatible_units/error
+Error: 1 and 1px are incompatible.
+  ,
+1 | a {b: atan2(1, 1px)}
+  |             ^ 1
+  |                === 1px
+  '
+  input.scss 1:13  root stylesheet
+
+<===>
+================================================================================
+<===> error/one_arg/input.scss
+a {b: atan2(0)}
+
+<===> error/one_arg/error
+Error: 2 arguments required, but only 1 was passed.
+  ,
+1 | a {b: atan2(0)}
+  |       ^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/invalid_arg/input.scss
+a {b: atan2($)}
+
+<===> error/syntax/invalid_arg/error
+Error: Expected identifier.
+  ,
+1 | a {b: atan2($)}
+  |              ^
+  '
+  input.scss 1:14  root stylesheet
```

Comment:
When testing error cases, it's a good idea to make sure that everything _except_ the error you're intending to test is valid so that you can be sure the code is failing due to the correct error. In this case, this is both invalid syntax _and_ only one argument.